.gitignore

```diff
@@ -42,6 +42,9 @@
 *.idb
 *.pdb
 
+# Test artifacts
+/lib/testing/test-tool
```

Comment:
nit: we have separate per-directory .gitignore files for the tools directories instead of putting entries into the top-level file; so maybe do the same here for consistency?

Reply:
Ack, will move it to a local .gitignore